packages/webpack/src/plugin.js

```diff
@@ -464,50 +464,72 @@ class LavaMoatPlugin {
                   )
                   return
                 }
-                diag.rawDebug(
-                  1,
-                  '> adding runtime (additionalChunkRuntimeRequirements)'
+                let runtimeChunks = []
+                if (
+                  chunk.name &&
+                  options.unlockedChunksUnsafe?.test(chunk.name)
```

Comment:
we trust this part can't be manipulated due to RegExp hardening? Any other attack scenarios we might be forgetting here?

Reply:
it _does_ say `unlockedChunksUnsafe` ... 😉

Reply:
yup, but it should only be unsafe to the selected chunk not the rest of the app. 

the attack would need to happen at build time, should not be possible to craft a package that messes with it without being part of the dependencies for webpack and its plugins/loaders etc.

And that's a concern of `@lavamoat/node` eventually. 

Am I missing anything? @weizman